routing/router.go

```diff
@@ -694,6 +694,12 @@ func (r *ChannelRouter) FindBlindedPaths(destination route.Vertex,
 		// Don't bother adding a route if its success probability less
 		// minimum that can be assigned to any single pair.
 		if totalRouteProbability <= DefaultMinRouteProbability {
+			log.Debugf("Not using route (%v) as a blinded "+
+				"path since it resulted in an low "+
+				"probability path(%.3f)",
+				route.ChanIDString(routeWithProbability.route),
+				routeWithProbability.probability,
+			)
```

Comment:
re my previous comment: ok cool - but then that code refactor should be done here. In the commit where it is currently done, it is a no-op